### PR TITLE
Define user environment variables

### PR DESCRIPTION
**Pull request by thiagorb**
Solves #2.

### CODE REVIEW

File: src/runner.c [hunk 1]
```diff
@@ -36,6 +48,14 @@ int main(int argc, char **argv)
 		new_argv[wrapper->argc] = NULL;
 	}
 
+	char *env_prefix = "SUID_WRAPPER_";
+	char *new_env[] = {
+		ssprintf("%sUID=%d", env_prefix, getuid()),
+		ssprintf("%sGID=%d", env_prefix, getuid()),
```
Comment (biocyberman) [inline src/runner.c]: Should it be `getgid()` instead of `getuid()` here?

File: src/runner.c [hunk 2]
```diff
@@ -36,6 +48,14 @@ int main(int argc, char **argv)
...
+		ssprintf("%sNAME=%s", env_prefix, getpwuid(getuid())->pw_name),
```
Comment (biocyberman) [inline src/runner.c]: The linux equivalent env var is `$USER`. Would it be good idea to use the same so it becomes `SUID_WRAPPER_USER` instead?

**Review by biocyberman — COMMENTED** [review]
tested with AD user, it works.

File: src/runner.c (hunk 1 above)
Comment (thiagorb) [reply src/runner.c]: Yep!

File: src/runner.c (hunk 2 above)
Comment (thiagorb) [reply src/runner.c]: You are right. I changed it to SUID_WRAPPER_USER.

**Comment by biocyberman** [comment]
Look neat with the new changes. Good to go for me.

**Pull request merged**